openedx/adg/common/lib/mandrill_client/client.py

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
Please rename `mandril_client` package to `mandril`

Reply:
I named it as you suggested but than github action was giving me some pylint issues, somehow it confuses our mandrill package with mandrill client.

Reply:
If  with `mandril` pylint is giving some issues, then either we have to go with `mandril_client` or we have to come up with a name which removes this `client` redundancy and also expresses the intent. Usman and I thought about different names like `madril_email_client` and a few others around this, but after all those names `mandril_client` seems better, so we are merging for now to remove the dependency. Zeeshan Arif when you'll back, if you still feel some improvement in the name is required, we can make it in some next iteration.